src/appleseed/renderer/modeling/phasefunction/henyeyphasefunction.cpp

```diff
@@ -110,10 +110,17 @@ class HenyeyPhaseFunction
             values->m_scattering_multiplier * values->m_scattering;
 
         // Ensure that extinction spectrum has unit norm, which is neccessary for distance sampling.
-        const float extinction_norm = max_value(values->m_precomputed.m_normalized_extinction);
+        double extinction_norm = 0.0;
+        const double power = 4.0;
```

Comment:
These are true constants so they should use CamelCase:

    const double Power = 4.0;
    const double RcpPower = 0.25;